actions-omitted.yaml

```diff
@@ -1,5 +1,16 @@
 # Avoid syncing certain actions to certain repos. (For example, go testing from non-go repos.)
 # This uses a glob prefix match (adds * to the end of each omit), so no need to specify '.yaml'
+knative/website:
```

Comment:
This repo has been archived, could we just remove it completely?

Reply:
Removed